src/libs/ReportUtils.js

```diff
@@ -974,7 +983,7 @@ function buildOptimisticChatReport(
         lastActionCreated: currentTime,
         notificationPreference,
         oldPolicyName,
-        ownerEmail,
+        ownerEmail: ownerEmail || CONST.REPORT.OWNER_EMAIL_FAKE,
```

Comment:
This change is needed because if we don't have it, when the owner email is null it creates some console errors